terraform/aks.tf

```diff
@@ -28,3 +28,7 @@ output "kube_config" {
   sensitive = true
   value     = azurerm_kubernetes_cluster.example.kube_config_raw
 }
+output "kube_context" {
```

Comment:
Why do you need to export kube_context?

Reply:
for this part: https://github.com/pachyderm/aml/blob/delimited-fixes/README.md#advanced-using-pachctl-locally

you need the context name to create the pachctl context if you have any existing pach config.